### PR TITLE
Fix markdown link and replacing the URL for create issues

### PR DESCRIPTION
**Pull request by rn4n**
Hello ✋

This PR fix the link for RFC 2119 and change the URL for creating new issues.

### CODE REVIEW

File: lang/pt-BR/index.md [hunk 1]
```diff
@@ -287,7 +286,7 @@ participação de:
 Toda colaboração na tradução pode ser acompanhada no link:
 http://pad.okfn.org/p/Fh9hjBPVu9
 
-Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/wendtecnologia/semver/issues).
+Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/semver/semver.org/issues).
```
Comment (alexandrtovmach) [inline lang/pt-BR/index.md]:
```suggestion
Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/semver/semver/issues).
```

Sorry I pointed you in a wrong way. There should be the same link as in original file, and it's `semver/semver`

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: lang/pt-BR/index.md (hunk 1 above)
Comment (rn4n) [reply lang/pt-BR/index.md]: No problem, fixed. 🙋‍♂️

**Review by alexandrtovmach — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
@rn4n thanks for contribution :+1:

**Pull request merged**